src/Meilisearch/ApiActions.cs

```diff
@@ -0,0 +1,85 @@
+namespace Meilisearch
+{
+    /// <summary>
+    /// List all available actions for a key.
+    /// </summary>
+    public static class ApiActions
```

Comment:
Is there a particular reason to not use an enum here?

Reply:
In C# enum cannot be mapped directly to string. We can use attributes to decorate them and extract the value of it. I think it's a bit hacky to be honest.